internal/toolkit/toolkit.go

```diff
@@ -145,6 +146,13 @@ func (d *DefaultToolkit) GetInput(name string) (string, bool) {
 	return s, ok
 }
 
+func (d *DefaultToolkit) GetInputList(name string) ([]string, bool) {
+	d.once.Do(d.init)
+	s, ok := d.Environment[fmt.Sprintf("INPUT_BLOCKED_NAMESPACES_%s", strings.ToUpper(name))]
```

Comment:
```suggestion
	s, ok := d.Environment[fmt.Sprintf("INPUT_%s", strings.ToUpper(name))]
```

This should be `"INPUT_%s" like `GetInput`